app/helpers/application_helper.rb

```diff
@@ -1219,7 +1219,10 @@ def allowed_filter_db?(db)
   end
 
   def miq_tab_header(id, active = nil, options = {}, &block)
-    content_tag(:li, :class => "#{options[:class]} #{active == id ? 'active' : ''}", :id => "#{id}_tab") do
+    content_tag(:li,
+                :class     => "#{options[:class]} #{active == id ? 'active' : ''}",
+                :id        => "#{id}_tab",
+                'ng-click' => "changeAuthTab('#{id}');") do
```

Comment:
@himdel It looks like this change was added to keep track of the current tab that user is on in the form, it was added by @AparnaKarve in 5b227b5bbc27302f4553e113aa990a29df355781 maybe she remembers if there was some other purpose behind adding that ng-click